### DISCUSSION TITLE
Can't verify email for Github Student Developer Package

### DISCUSSION THREAD

**Discussion by catches** [community/community#51898] [GitHub Education]
Hey, so I'm trying to apply for the Github Student Package, however, I can't verify my email because my school gmail prevents me from receiving emails from other sites, is there a way I can still apply for the Github Student Package? Thanks.

**Comment by samus-aran** [comment]
Hi @catches 

Have you tried reaching out to your institute  IT department and explain the situation. GitHub requires your email to be verified and if they were able to add GitHub domain to their whitelist that would assist you to progress on to the next verification step.
